aws/resource_aws_route53_record.go

```diff
@@ -235,10 +235,9 @@ func resourceAwsRoute53Record() *schema.Resource {
 			},
 
 			"allow_overwrite": {
-				Type:       schema.TypeBool,
-				Optional:   true,
-				Computed:   true,
-				Deprecated: "The next major version of the Terraform AWS Provider will require importing existing records",
+				Type:     schema.TypeBool,
```

Comment:
Where there any tests removed when the attribute was deprecated that should be restored?

Reply:
Nope we don't tend to remove testing on deprecations as the functionality is still expected to work as it did previously. 👍 Here's the deprecation pull request for verification: #7734